Calculator/main.js

```diff
@@ -0,0 +1,126 @@
+class Calculator {
+constructor(previousOperandTextElement, currentOperandTextElement) {
+    this.previousOperandTextElement = previousOperandTextElement
+    this.currentOperandTextElement = currentOperandTextElement
+    this.clear()
+}
+
+clear() {
+    this.currentOperand = ''
+    this.previousOperand = ''
+    this.operation = undefined
+}
+
+delete() {
+    this.currentOperand = this.currentOperand.toString().slice(0, -1)
+}
+
+appendNumber(number) {
+    if (number === '.' && this.currentOperand.includes('.')) return
+    this.currentOperand = this.currentOperand.toString() + number.toString()
```

Comment:
this.currentOperand = \`${this.currentOperand}${number}\`;

Reply:
fixed